packages/web-api-scan-job-manager/src/batch/job-task.spec.ts

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { JobTask } from './job-task';
+
+describe(JobTask, () => {
+    let jobTask: JobTask;
+
+    it('id should include correlation value', () => {
+        const correlationId = `correlationId_${new Date().valueOf()}`;
```

Comment:
>{new Date().valueOf()} [](start = 46, length = 22)

avoid using changing values for test

Reply:
this is the same in the old package, I'd like to focus on what was changed first for the sake of time